venia-integration-tests/run-tests.js

```diff
@@ -77,21 +77,25 @@ const dockerRuns = {};
 const port = new URL(baseUrl).port;
 
 let dockerCommand = null;
+let dockerImage = 'cypress/included:8.3.1';
+if (process.env.DockerRegistry) {
```

Comment:
All other uses of process.env are capitalized in this repo. Might be a good idea to keep it consistent.